<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Book</class>
 <widget class="QWidget" name="Book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>خرید بلیط </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>E:/shopping-cart.png</normaloff>E:/shopping-cart.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 255);</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>681</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>2  Yekan</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>: اسم </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="font">
       <font>
        <family>2  Yekan</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>2  Yekan</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>: اتوبوس</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="font">
       <font>
        <family>2  Yekan</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>2  Yekan</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>: صندلی</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="font">
       <font>
        <family>2  Yekan</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>2  Yekan</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 183, 65);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>خرید بلیط</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 183, 65);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>E:/previous.png</normaloff>E:/previous.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
